src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,49 @@
+package duke.task;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Creates a task with a description and marked as not done by default.
+     * @param description Description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Creates a task with a description and done status.
+     * @param description Description of task.
+     * @param isDone Done status of task.
+     */
+    protected Task(String description, boolean isDone) {
```

Comment:
Consider setting this constructor as private - I suppose this is called when setting a task as done? The method can be protected but the constructor itself can be private.